### PR TITLE
Django 1.7 update.

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

File: test_utils/fixture_tables.py [hunk 1]
```diff
@@ -2,15 +2,19 @@
...
-
 from django.conf import settings
 from django.core import serializers
-from django.db import router, DEFAULT_DB_ALIAS
+from django.db import DEFAULT_DB_ALIAS, router
 from django.db.models import get_apps
-from django.utils.itercompat import product
+
+try:
+    from itertools import product
```
Comment (davedash) [inline test_utils/fixture_tables.py:16]: Couldn't you always use `itertools.product`?  Or is that only in certain python versions?

Comment (robhudson) [inline test_utils/fixture_tables.py:16]: It was added in Python 2.6, which is why Django had a copy of it for older Python versions. I don't know how far back test-utils supports.

Comment (davedash) [inline test_utils/fixture_tables.py:16]: Fair enough.

Do you mind adding a comment stating that product is in python 2.6+?

That way if we _do_ decide to stop supporting <2.6 we can.

**Comment by robhudson** [comment]
Updated based on feedback from Mr. @davedash

**Pull request merged**